RobotFramework_TestsuitesManagement/Config/CConfig.py

```diff
@@ -522,7 +522,8 @@ def __loadConfigFileLevel2(self):
             raise Exception
         if self.sConfigName not in oSuiteConfig:
             CConfig.sLoadedCfgError = f"Testsuite management - Loading configuration level 2 failed! \n \
-        The variant '{self.sConfigName}' is not defined in '{os.path.abspath(self.sTestSuiteCfg)}'"
+        The variant '{self.sConfigName}' is not defined in '{os.path.abspath(self.sTestSuiteCfg)}' \n \
+        Please find the suitable variant in this list: {list(oSuiteConfig.keys())}"
```

Comment:
Hi Son,
very good!
small findings:
- single quotes per entry are missing
- if the variant names are many and long its difficult to read, therefore please align properly with other content of the log file and log possible variants line by line.

Pleaes check how TML behaves as reference.

Just trigger a BVT with a wrong variant name.

Thank you,
 Thomas